js/core/utils/data.js

```diff
@@ -126,11 +127,17 @@ var combineGetters = function(getters) {
             path = name.split(".");
             last = path.length - 1;
 
-            for(i = 0; i < last; i++) {
-                current = current[path[i]] = {};
+            for(i = 0; i <= last; i++) {
```

Comment:
What about moving the assignment outside the loop?
```
path = name.split(".");
last = path.length - 1;
if(last < 0) {
    return;
}

for(i = 0; i < last; i++) {
    pathItem = path[i];
    current = pathItem in current
        ? current[pathItem]
        : current[pathItem] = {};
}

current[path[last]] = value;
```

Reply:
Moved.

Reply:
:ok_hand: